_articles/open-firmware-smmstore.md

```diff
@@ -0,0 +1,66 @@
+```
+
+```
+calling Schedule method failed: "failed to add boot entry: exit code: 5
+```
+
+```
+system76-firmware: failed to schedule: failed to add boot entry: exit code: 5
+```
+
+we may need to clear the SMMSTORE ROM by flashing new firmware using a flash drive with a custom firmware update tool. Here are the steps to perform that action. After flashing, this will remove any custom EFI variables (such as those installed by boot managers). We have information on repairing the bootloader at the end of this article. 
+
+Please download the zip file for your sytem and unzip the files within to a FAT32 formatted USB drive. Then shutdown the laptop and remove all NVMe M.2 drive(s) in the system using the step by step instructions for your system:
+
+|   System            |              Patched Firmware                            |                                        Tech Docs                                            |
+|:-------------------:|:--------------------------------------------------------:|:-------------------------------------------------------------------------------------------:|
+| Bonobo WS  (bonw14) | [bonw14-76ec.zip](/files/clear-smmstore/bonw14-76ec.zip) | [bonw14](https://tech-docs.system76.com/models/bonw14/repairs.html#replacing-an-m2nvme-ssd) |
+| Darter Pro (darp6)  | [darp6-76ec.zip](/files/clear-smmstore/bonw14-76ec.zip)  | [darp6](https://tech-docs.system76.com/models/bonw14/repairs.html#replacing-an-m2nvme-ssd)  |
+| Darter Pro (darp7)  | [darp7-76ec.zip](/files/clear-smmstore/bonw14-76ec.zip)  | [darp7](https://tech-docs.system76.com/models/bonw14/repairs.html#replacing-an-m2nvme-ssd)  |
+| Galago Pro (galp4)  | [galp4-76ec.zip](/files/clear-smmstore/bonw14-76ec.zip)  | [galp4](https://tech-docs.system76.com/models/bonw14/repairs.html#replacing-an-m2nvme-ssd)  |
+| Galago Pro (galp5)  | [galp5-76ec.zip](/files/clear-smmstore/bonw14-76ec.zip)  | [galp5](https://tech-docs.system76.com/models/bonw14/repairs.html#replacing-an-m2nvme-ssd)  |
+| Lemur Pro (lemp9)   | [lemp9-76ec.zip](/files/clear-smmstore/bonw14-76ec.zip)  | [lemp9](https://tech-docs.system76.com/models/bonw14/repairs.html#replacing-an-m2nvme-ssd)  |
+| Lemur Pro (lemp10)  | [lemp10-76ec.zip](/files/clear-smmstore/bonw14-76ec.zip) | [lemp10](https://tech-docs.system76.com/models/bonw14/repairs.html#replacing-an-m2nvme-ssd) |
+| Oryx Pro (oryp6)    | [oryp6-76ec.zip](/files/clear-smmstore/bonw14-76ec.zip)  | [oryp6](https://tech-docs.system76.com/models/bonw14/repairs.html#replacing-an-m2nvme-ssd)  |
+
+With all drive(s) removed, replace the cover and boot with the USB stick inserted. After the firmware has been flashed, remove the USB stick and reinstall the drive(s).
+
+Once the system is put back together we'll need to boot with a live image in order to repair the boot loader. We provide step by step instructions on how to do this in our support articles in the links below:
```

Comment:
Booting into a live image should only be necessary if `\EFI\BOOT\BOOTX64.EFI` loads the Windows Boot Manager. The grub/systemd-boot menus will continue to work if they can be accessed.

Reply:
This is what I have "Once the system is put back together we may need to boot with a live image in order to repair the boot loader if you dual-booting with Windows 10. We provide step by step instructions on how to do this in our support articles in the links below:"

Reply:
nit: 

>  if you dual-booting with

should be

>  if you dual-boot with

Reply:
Fixed: https://github.com/system76/docs/pull/561/commits/a52f1800a8fc029a568abd5f2cc37ea216dd0a06